test/libyul/YulOptimizerTestCommon.cpp

```diff
@@ -62,7 +62,7 @@
 #include <libyul/backends/evm/EVMDialect.h>
 #include <libyul/backends/evm/EVMMetrics.h>
 #include <libyul/AsmAnalysis.h>
-#include <libyul/CompilabilityChecker.h>
```

Comment:
even if we wanted to name it compatibility checker (which we don't), it would still require to move the file and edit the cmakelists.